azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -978,6 +980,10 @@ public String submitExecutableFlow(final ExecutableFlow exflow, final String use
             throw new ExecutorManagerException("Flow " + flowId
                 + " is already running. Skipping execution.",
                 ExecutorManagerException.Reason.SkippedExecution);
+          } else if (running.size() > AZKABAN_MAX_NUM_CONCURRENT_FLOW) {
+            throw new ExecutorManagerException("Flow " + flowId
+                + " has more than 30 concurrent runs. Skipping",
```

Comment:
30 in log message should be replace by AZKABAN_MAX_NUM_CONCURRENT_FLOW.

Reply:
Done.